src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_common.py

```diff
@@ -354,3 +355,49 @@ def flexible_server_log_list(client, resource_group_name, server_name, filename_
         files.append(f)
 
     return files
+
+
+def flexible_server_version_upgrade(cmd, client, resource_group_name, server_name, version, yes=None):
+    if not yes:
```

Comment:
Looks like in the thread, Portal team wrote this.
1.	Don’t show Upgrade for Burstable sku and existing 8.0 servers.
2.	Send 8.0.21 until backend supports to accept major version (Ex: 8.0)
3.	Move feature switch to subscription level
4.	Use 2021-12-01-preview to upgrade the server.

Are we allowing Burstable server for version upgade? Or do we need to filter that out as well? Wanted to make sure if you already had discussion with the feature owner about these agenda

Reply:
1. If you see the next `if` block, I'm validating that `sku.tier` is not `Burstable`. Also, I'm validating that the new version is strictly greater than the current one.
2. I'm sending 8.0.21 for MySQL version 8.
4. Yes, I'm using 2021-12-01.